README_el.md

```diff
@@ -1,11 +1,11 @@
 # ikra v1.2.0 -RNAseq pipeline centered on Salmon-<img src="img/ikra.png" width="20%" align="right" />
 
 
-[idep](http://bioinformatics.sdstate.edu/idep/)のinputとして発現量テーブル（gene × sample）をexperiment matrixから自動でつくる。salmonを用いる。
+An expression amount table (gene × sample) is automatically created from the experiment matrix as an input of [idep](http://bioinformatics.sdstate.edu/idep/). This tool uses [salmon](https://combine-lab.github.io/salmon/).
 
-## 重要　bugについて　2019/04/30
+## Important　about bug　2019/04/30
 
-ikraの`tximport_R.R`にサンプルを取り違えうる重大なバグが見つかり、修正しました。最新版に更新してお使いください。古いバージョンを使われていた方は、中間ファイルは問題ありませんので、`output.tsv`およびexperiment matrixと同じディレクトリにコピーされている`tximport_R.R`を削除し、もう一度新しいikra.shを実行してください。大変ご迷惑をおかけいたしました。
+A serious bug has been found and fixed that could misinterpret a sample in the `tximport_R.R` of ikra.Please update it to the latest version.If you are using the old version, there is no problem with the intermediate file, so delete `tximport_R.R` copied to the same directory as` output.tsv` and experiment matrix, and execute new ikra.sh again. please.We apologize for the inconvenience.
```

Comment:
A serious bug was found in the `tximport_R.R` and fixed. Please update it to the latest version. If you are using the old version(<1.2.0), there is no problem with the intermediate file, so delete `tximport_R.R` copied to the same directory as` output.tsv` and experiment matrix, and execute new ikra.sh again. We apologize for the inconvenience.